src/components/SortButton.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import SortButton from './SortButton'
+
+describe('SortButton', () => {
+  it('renders correctly', () => {
+    const tree = renderer.create(<SortButton />)
+    expect(tree).toMatchSnapshot()
```

Comment:
If I'm not mistaken, you can simplify this test. Try the following: 
`test('renders correctly', () => {
expect(SortButton).toMatchSnapshot()
})`